core/lib/zksync_core/src/eth_sender/eth_tx_aggregator.rs

```diff
@@ -293,9 +293,9 @@ impl EthTxAggregator {
     }
 
     /// Loads current verifier config on L1
-    async fn get_recursion_scheduler_level_vk_hash<E: BoundEthInterface>(
+    async fn get_recursion_scheduler_level_vk_hash(
         &mut self,
-        eth_client: &E,
+
```

Comment:
Probably you don't want an empty line

Reply:
TIL about one more weird `rustfmt` default. Will fix, thanks!